ReactAndroid/src/androidTest/js/ScrollViewTestModule.js

```diff
@@ -37,54 +36,52 @@ class Item extends React.Component {
   }
 }
 
-var getInitialState = function() {
-  var data = [];
-  for (var i = 0; i < NUM_ITEMS; i++) {
-    data[i] = {text: 'Item ' + i + '!'};
+class ScrollViewTestApp extends React.Component {
+  constructor() {
+    super();
+    this.scrollView = React.createRef();
```

Comment:
nit: We should just use class properties for this.

```JavaScript
class ScrollViewTestApp extends React.Component {
  scrollView = React.createRef();
  //...
```

This way, we won't have to implement our own constructor.